Source/MonoGame.Extended/FramesPerSecondCounter.cs

```diff
@@ -24,7 +25,7 @@ public void Reset()
             _sampleBuffer.Clear();
         }
 
-        public void Update(float deltaTime)
+        public void UpdateFPS(float deltaTime)
```

Comment:
Does this have to be called `UpdateFPS`? Can't we just have another overload of `Update`?